tests/test_profiler.py

```diff
@@ -1,7 +1,19 @@
-from pytorch_lightning.profiler import Profiler, AdvancedProfiler
 import time
+
 import numpy as np
 
+from pytorch_lightning.profiler import Profiler, AdvancedProfiler
+
+
+def _assert_almost_equal(got, expect, tol=0.1):
```

Comment:
Is there a good reason not to use: numpy.testing.assert_allclose?

Reply:
I did not know it lol